src/objective-c/GRPCClient/GRPCCall.h

```diff
@@ -155,6 +155,18 @@ typedef NS_ENUM(NSUInteger, GRPCErrorCode) {
 };
 
 /**
+ * Flags for options of a gRPC call
```

Comment:
I'm not sure I support binding gRPC semantics to HTTP directly. There's very real possibilities of non-HTTP transports existing in the future, and with them we might explore different semantics.

Better to outline what things mean for gRPC here.